homeassistant/components/risco/config_flow.py

```diff
@@ -95,7 +128,32 @@ async def async_step_user(self, user_input=None):
                 return self.async_create_entry(title=info["title"], data=user_input)
 
         return self.async_show_form(
-            step_id="user", data_schema=DATA_SCHEMA, errors=errors
+            step_id="cloud", data_schema=CLOUD_SCHEMA, errors=errors
+        )
+
+    async def async_step_local(self, user_input=None):
+        """Configure a local based alarm."""
+        errors = {}
+        if user_input is not None:
+            await self.async_set_unique_id(user_input[CONF_HOST])
```

Comment:
Host is not a valid unique id.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-id-requirements

Reply:
ok. I'll change it to the site id, but that requires connecting and querying for it, which in most cases would make this meaningless as Risco usually does not accept another connect (and in many cases it might just get stuck).

This also leads me to a different question I don't know where to ask:
Because we're connecting from the config flow, and then connecting again when setting up the platforms, Risco sometimes gets stuck. Adding `await asyncio.sleep(1)` before creating the config entry solves this, but I don't like it, and I definitely don't want tests to have to wait for it. Any suggestions how to resolve?

Reply:
Store the sleep time in a constant to make it easy to patch to 0 in tests.